tests/regression/56-floats/06-library_functions.c

```diff
@@ -0,0 +1,40 @@
+// PARAM: --enable ana.int.interval --enable ana.int.def_exc --enable ana.float.interval
+#include <assert.h>
+#include <math.h>
+
+int main() {
+    double dbl_min = 2.2250738585072014e-308;
+    double inf = 1. / 0.;
+    double nan = 0. / 0.;

```

Comment:
Again, i prefer the c standard macros/functions instead of magic computations. What about `NAN`
? 
Once we can actually say something meaningful about nan, we should even test at least two different kind of nans...

Reply:
Again, If I do the regression test does not work anymore

Reply:
here the problem seems, that the macros "NAN" and "INFINITY" call some other `__builtin_` functions that are not implemented yet

Reply:
That sounds reasonable